modules/flowable-task-service/src/main/resources/org/flowable/task/service/db/mapping/entity/HistoricTaskInstance.xml

```diff
@@ -996,6 +996,12 @@
       <if test="processInstanceId != null">
         and ${queryTablePrefix}PROC_INST_ID_ = #{processInstanceId}
       </if>
+    <if test="taskIdList != null &amp;&amp; taskIdList.size() &gt; 0">
```

Comment:
We recently did a change when checking for a size of a list. please use `!taskIdList.empty` instead.